community/mopidy/APKBUILD

```diff
@@ -0,0 +1,79 @@
+# Contributor: Łukasz Jendrysik <scadu@yandex.com>
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=mopidy
+_pkgname=Mopidy
+pkgver=2.2.0
+pkgrel=0
+pkgdesc="Mopidy is an extensible music server"
+url="http://www.mopidy.com/"
+arch="noarch"
+license="Apache-2.0"
+depends="python py-pykka py-tornado gst-plugins-base gst-plugins-good
+	gst-plugins-ugly py-gst py-setuptools py-libxml2 py-requests
+	py-six
+	"
+depends_dev=""
+makedepends="python-dev py2-sphinx"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages="$pkgname-doc"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
+	mopidy.conf
+	logging.conf
+	mopidy.initd
+	"
+
+_builddir="$srcdir"/$_pkgname-$pkgver
+
+pkgusers="mopidy"
+pkggroups="audio"
+
+build() {
+	cd "$_builddir"
+	python setup.py build
+
+	make -C docs SPHINXBUILD=sphinx-build-2 man
+	make -C docs SPHINXBUILD=sphinx-build-2 html
+}
+
+package() {
+	cd "$_builddir"
+	python setup.py install --prefix=/usr --root="$pkgdir"
+	install -Dm755 extra/mopidyctl/mopidyctl "${pkgdir}/usr/bin/mopidyctl"
+
+	install -dm755 "${pkgdir}/usr/share/doc/mopidy"
+	cp -r docs/_build/html "${pkgdir}/usr/share/doc/mopidy"
+
+	install -dm755 "${pkgdir}/usr/share/man/man1/"
+	gzip -c docs/_build/man/mopidy.1 > "${pkgdir}/usr/share/man/man1/mopidy.1.gz"
+
+	install -dm755 "${pkgdir}/usr/share/man/man8/"
+	gzip -c extra/mopidyctl/mopidyctl.8 > "${pkgdir}/usr/share/man/man8/mopidyctl.8.gz"
+
+	install -Dm644 "${srcdir}/mopidy.conf" \
+		"${pkgdir}/etc/mopidy/mopidy.conf"
+	install -Dm644 "${srcdir}/logging.conf" \
+		"${pkgdir}/etc/mopidy/logging.conf"
+
+	install -Dm755 "$srcdir/$pkgname.initd" \
+		"$pkgdir/etc/init.d/$pkgname"
+
+	install -Dm644 LICENSE "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+
+	install -dm755 -o mopidy -g audio \
+		"${pkgdir}/etc/mopidy/extensions.d" \
+		"${pkgdir}/var/log/mopidy" \
+		"${pkgdir}/var/cache/mopidy" \
+		"${pkgdir}/var/lib/mopidy/media" \
+		"${pkgdir}/var/lib/mopidy/playlists"
+}
+
+check() {
+	cd "$_builddir"
+	python -m mopidy --help > /dev/null
```

Comment:
Just calling the help function does not suffice as a check function. If there is no test suite, the preference is to mark this package as `!check`.